locales/pt/pt.json

```diff
@@ -358,11 +358,11 @@
     "Mongolia": "Mongólia",
     "Montenegro": "Montenegro",
     "Month To Date": "Mês Até À Data",
-    "Monthly": "Monthly",
-    "monthly": "monthly",
+    "Monthly": "Mensal",
+    "monthly": "Mensal",
```

Comment:
The word "month" is used in context. Is it correct that you indicate it with a capital letter?

Reply:
Indeed, the word should have started with a lower case letter.